pkg/manifestgen/manifest.go

```diff
@@ -24,6 +24,8 @@ import (
 	securejoin "github.com/cyphar/filepath-securejoin"
 )
 
+const GenWarning = "# This manifest was generated by flux bootstrap. DO NOT EDIT."
+
```

Comment:
```suggestion
const GenWarning = `# This manifest was generated by flux bootstrap. DO NOT EDIT.
# For modifications to the Flux configuration, see the "customize Flux" documentation.
# Ref: https://fluxcd.io/docs/installation/#customize-flux-manifests
`

```

Reply:
What about Terraform? this comment ends up everywhere manifestgen is used

Reply:
I think we need to have a dedicated page for "Customize Flux manifests" where we address both the CLI and TF bootstrap options. After that we could update this comment and point to the new page.

Reply:
We should record this in a follow-up issue.